src/app/shared/session/session.query.ts

```diff
@@ -47,6 +48,6 @@ export class SessionQuery extends Query<SessionState> {
   generateGitHubAppInstallationUrl(entryType: EntryType): string {
     let queryParams = new HttpParams();
     queryParams = queryParams.set('state', entryType);
-    return 'https://github.com/apps/dockstore/installations/new?' + queryParams;
+    return Dockstore.GITHUB_APP_INSTALLATION_URL + '/installations/new?' + queryParams;
```

Comment:
In src/app/github-name-to-id.pipe.ts you use the `HttpParams.toString()` method; seems clearer and you should probably do it here... I didn't even know this would work like this.